### DISCUSSION TITLE
Please let us pin the old feed or provide a way to opt out

### DISCUSSION THREAD

**Discussion by hodgef** [community/community#53749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I use the feed to follow my project's github-actions and activity regarding my projects. Please do not default to the cluttered/social view without a way to opt out. It's a bad experience for users who have a similar use-case (maintainers) and results in needless disruption.

Do we really need to spend time coding a user script to click the Following tab? I don't see why you'd want to bother devs (your evangelists) in this way. Team effort would be better spent on developing enterprise features.

Happy to discuss in Teams about this. Thanks!

**Comment by YannCebron** [comment]
I don't want a social network that let's me "explore GREAT things". I want to get my work done by selecting explicitly what I want to follow. Thanks.

**Comment by tansanDOTeth** [comment]
Yes, I agree. The social aspect is very unproductive. I would really prefer to not use any social features and only see github as a productivity tool and a place to focus.

**Comment by hodgef** [comment]
I ended up adding a userscript to my browsers. It's quite annoying since I have to add it to every computer.
https://gist.github.com/hodgef/87815c4df3a221551156aca81a32a33b

I would encourage the GH org folks to reconsider this kind of rollout. It's not a good look.

**Reply by kmk3** [reply]
> I ended up adding a userscript to my browsers. It's quite annoying since I
> have to add it to every computer.

Related: The feed can also be hidden with a uBlock Origin filter:

* <https://github.com/orgs/community/discussions/53712#discussioncomment-5732926>

**Reply by yan12125** [reply]
Thanks for this userscript. uBlock Origin filters remove the "For you" tab but does not make "Following" the default.

**Reply by yan12125** [reply]
The userscript does not work reliably on ViolentMonkey. I need to also specify run-at:
```
// @run-at       document-idle
```

**Comment by PennRobotics** [comment]
A lot of people don't want to see recommended repositories. Not enough developers are focusing on the inverse problem:

**I don't want to be _discovered_ or _promoted_ or _highly visible_.**

I don't want my repositories&mdash;even though they are public&mdash;to get the attention of random or even tangentially relevant people, unless I specifically link to a repository or someone has a very specific Google search where my repo is mysteriously the only/best resource available.

Many of my repositories are half-baked explorations or hobbies&mdash;definitely not end products or giant team efforts. I have a family and don't take time to document code or finish features or properly debug/sanitize/optimize/test. It is my expectation that an algorithmically recommended viewer of my repository is more likely to create stress (via issues, pull requests, external linking, whatever) than to alleviate it.

In the same vein... In some places, it's legal to stand on a neighborhood sidewalk and stare through the windows of a stranger's house and even to yell onto their property, but that almost universally would upset the house owner.

Specifically, I want the option to _opt out of being someone's recommendation_.

**Reply by cristoffer** [reply]
I fully agree.

I have repositories that are quite old and not up to my current standards. I had never thought of removing them before because of sentiment. Now i will definitely remove them because I don't want anyone to link that code to me or my current performance. 
I will also not create public repos on GitHub anymore, the trust is gone.

**Reply by yan12125** [reply]
Just a guess - setting your profile to private may achieve what you want.

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/setting-your-profile-to-private

**Comment by Polikarpch** [comment]
You speak very logically.

**Comment by Polikarpch** [comment]
Thanks for this userscript. uBlock Origin filters remove the "For you" tab but does not make "Following" the default.
